internal/injector/config/config.go

```diff
@@ -26,16 +26,19 @@ type Config interface {
 	visit(Visitor, string) error
 }
 
+type PackageLoader = func(context.Context, string, ...string) ([]*packages.Package, error)
```

Comment:
```suggestion
type PackageLoader func(context.Context, string, ...string) ([]*packages.Package, error)
```
Any reason to use the equal form here ?

Reply:
I think of it just as a short-hand for the function type... I don't want to force casting non-literal functions to it...